dipy/nn/cnn_1denoiser/cnn_1denoiser.py

```diff
@@ -0,0 +1,283 @@
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras.layers import Conv1D, Activation
+from tensorflow.keras.models import *
+from tensorflow.keras.layers import Input
+#from patchify import patchify, unpatchify
+from sklearn.model_selection import train_test_split
+from datetime import datetime
+now = datetime.utcnow().strftime("%Y%m%d%H%M%S")
+root_logdir = "tf_logs"
+logdir = "{}/run-{}/".format(root_logdir, now)
+'''
+Title : Denoising diffusion weighted imaging data using CNN
+=====
+
+we constructed a simple 1D-CNN model that has five layers, including two convolutional layers, each
+followed by a max-pooling layer, and a dense layer (Figure 1). The first convolutional layer was given an
```

Comment:
Please remove the reference to the figure (is this just a copy paste of the paper abstract? I'd suggest rewriting if so)